src/dotnet-aspnet-codegenerator/DotNetBuildCommandHelper.cs

```diff
@@ -40,9 +39,7 @@ internal class DotNetBuildCommandHelper
 
             var command = Command.CreateDotNet(
                     "build",
-                    args,
-                    framework,
-                    configuration: configuration)
```

Comment:
why remove the configuration?

Reply:
it was not being used. couple more minor edits like this.